CHANGELOG.md

```diff
@@ -55,6 +55,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Metroid Dread
 
 - Fixed: The option to hide scans with Nothing data now behaves as expected.
+- Fixed: Starting with more than 3 Energy Parts lets you able to export now.
```

Comment:
Wording seems off. I'd say something like "It is now possible to start the game with more than 3 Energy Parts.

Reply:
"start the game" sounds wrong to me. you couldnt even *attempt* to start before because you couldnt export.
i was considering using "... doesn't throw an exception during export" but thats too much tech jargon.

Reply:
I would say `Fixed: Seeds with more than 3 starting Energy Parts can be exported and played`

Though i'm not really sure why players would want to start with 5 EParts and not 1 tank and 1 part.

Reply:
i assume the likeleist way fro this get reprod is by someone wanting to start with all E and all Eparts.